apps/etterlatte-saksbehandling-ui/client/src/components/person/brev/LastOppBrev.tsx

```diff
@@ -7,22 +7,41 @@ import { useApiCall } from '~shared/hooks/useApiCall'
 import { DokumentVisningModal, PdfVisning } from '~shared/brev/pdf-visning'
 import { isPending } from '~shared/api/apiUtils'
 import { ISak } from '~shared/types/sak'
+import { round } from 'lodash'
+
+/**
+ * Husk å endre [proxy_body_size] i nais-filene hvis du skal øke maks filstørrelse.
+ * Mer info: https://docs.nais.io
+ **/
+const MAKS_FILSTOERRELSE_MB = 1
 
 export const LastOppBrev = ({ sak }: { sak: ISak }) => {
   const navigate = useNavigate()
 
   const [isOpen, setIsOpen] = useState(false)
   const [filURL, setFilURL] = useState<string | undefined>()
   const [valgtFil, setValgtFil] = useState<File | undefined>()
+  const [error, setError] = useState<string>()
 
   const [lastOppStatus, lastOppBrev] = useApiCall(opprettBrevFraPDF)
 
   const onFileChange = (event: any) => {
     const fil = event.target.files[0]
-    setValgtFil(fil)
 
-    const url = URL.createObjectURL(fil)
-    setFilURL(url)
+    const filstoerrelseMegabytes = fil.size / 1000000
```

Comment:
```suggestion
    const filstoerrelseMegabytes = fil.size / (1024 * 1024)
```
antar fil.size gir bytes

Reply:
Hadde det til å begynne med, men da fikk jeg forskjellig resultat i OSX Finder og frontend sin utregning. 🤔 

...men eh, la oss bare kjøre på!